core/injector/application_injector.hpp

```diff
@@ -581,8 +579,8 @@ namespace kagome::injector {
         di::bind<consensus::grandpa::Gossiper>.template to<network::GossiperBroadcast>(),
         di::bind<network::Gossiper>.template to<network::GossiperBroadcast>(),
         di::bind<network::SyncClientsSet>.to(std::move(get_sync_clients_set)),
-        di::bind<network::SyncProtocolClient>.template to<consensus::SynchronizerImpl>(),
-        di::bind<network::SyncProtocolObserver>.template to<consensus::SynchronizerImpl>(),
+        di::bind<network::SyncProtocolClient>.template to<network::RemoteSyncProtocolClient>(),
```

Comment:
I guess it can be removed. Actually I am not sure how it will work, considering that we should create several remote sync protocol clients. So not clear which one will be created when we call injector.create

Reply:
Agree. Done.